graylog2-server/src/test/java/org/graylog/testing/completebackend/DefaultMavenProjectDirProvider.java

```diff
@@ -27,6 +27,11 @@ public Path getProjectDir() {
         return getProjectReposDir().resolve("graylog2-server");
     }
 
+    @Override
+    public Path getBinDir() {
+        return getProjectReposDir().resolve("graylog-plugin-enterprise/enterprise/bin");
```

Comment:
Is it OK to reference the enterprise plugin here, @janheise ?

Reply:
@todvora probably not, thank you. I changed it to a more sensible location (graylog2-server/bin, which is the default in graylog.conf) but during our maven packaging, no binaries are written there. But the feature is mainly for our enterprise tests so I guess it should be fine.